library/Ivoz/Provider/Domain/Service/BillableCall/CsvExporter.php

```diff
@@ -99,6 +91,48 @@ public function execute(
             $criteria['_timezone'] = $brand->getDefaultTimezone()->getTz();
         }
 
+        $direction = $scheduler
+            ? $scheduler->getCallDirection()
+            : null;
```

Comment:
The original code has `BillableCallInterface::DIRECTION_OUTBOUND` as default value for $direction.
Shouldn't be more backwards compatible using that default value?

```suggestion
        $direction = $scheduler
            ? $scheduler->getCallDirection()
            : BillableCallInterface::DIRECTION_OUTBOUND;
```

Although I think it doesn't really matter because that parameter is always included as value or null in all method invokations.

Reply:
I'm thinking about making scheduler required argument. This services is only used from CallCsvReport\CsvAttacher and there is no way to create a report without a scheduler (It turns out that I'm allowing updates on client API, I'll fix it in another PR). Do you agree?

Reply:
Ok, making scheduler non optional will simplify all ternary operators here.